website/content/en/preview/concepts/metrics.md

```diff
@@ -97,6 +97,12 @@ Duration of cloud provider method calls. Labeled by the controller, method name
 ### `karpenter_cloudprovider_instance_price_estimate`
```

Comment:
```suggestion
### `karpenter_cloudprovider_instance_type_price_estimate`
```

Maybe change this so that we align all of these under the `instance_type` namespace

Reply:
done